### ISSUE TITLE
Millenium Falcon Title Segnor Loop Card Missing

### ISSUE THREAD

**Issue by ankyle** [tjakubo2/TTS_xwing#44]
There are two Millenium Falcon title cards, one for evade action and one for Segnor Loop. When you load a squad with the Segnor Loop title it loads the evade title and dial does not include the Segnor Loop. http://xwing-miniatures.wikia.com/wiki/Millennium_Falcon_(HOR)  

**What was supposed to happen:**
List should load and maybe give an option for both titles(much like the pilot cards), or load correct card

**How to make this happen:**
Programming magic fingers

**Does this still occur after saving & loading game:**
N/A

**Comment by tjakubo2** [comment]
Hey, I know this isn't most intuitive way to have it done, but it's explained on point 8 of squad building docs: https://github.com/tjakubo2/TTS_xwing/wiki/Doc:-Squad-Building#squad-builder-option

In short, press number keys on mouseover or right click -> Set State to change versions. Right click -> Reset will lock the card in the current state if you want.  
Sadly they do have technically the same name and there's little space to do the choosing-interface like with pilot cards. I thought about printing a warning when you iclude a card like that, feel free to speak up if you have any suggestions :)

**Comment by ankyle** [comment]
Thanks for the quick response, I didn't know about the mouse over numbers. Is there an easy way to assign the segnorn loop dial maneuver?

I was looking at the code and wish I could contribute but it looks way above my head.
Sent from my mobile phone

On Aug 5, 2017, at 7:48 AM, tjakubo2 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey, I know this isn't most intuitive way to have it done, but it's explained on point 8 of squad building docs: https://github.com/tjakubo2/TTS_xwing/wiki/Doc:-Squad-Building#squad-builder-option

In short, press number keys on mouseover or right click -> Set State to change versions. Right click -> Reset will lock the card in the current state if you want.
Sadly they do have technically the same name and there's little space to do the choosing-interface like with pilot cards. I thought about printing a warning when you iclude a card like that, feel free to speak up if you have any suggestions :)

**Comment by ankyle** [comment]
Ignore that last question, I actually read the card. No need for the maneuver.

Sent from my mobile phone

On Aug 5, 2017, at 7:48 AM, tjakubo2 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey, I know this isn't most intuitive way to have it done, but it's explained on point 8 of squad building docs: https://github.com/tjakubo2/TTS_xwing/wiki/Doc:-Squad-Building#squad-builder-option

In short, press number keys on mouseover or right click -> Set State to change versions. Right click -> Reset will lock the card in the current state if you want.
Sadly they do have technically the same name and there's little space to do the choosing-interface like with pilot cards. I thought about printing a warning when you iclude a card like that, feel free to speak up if you have any suggestions :)

**Comment by tjakubo2** [comment]
Since it's a thing you can decide for after doing a maneuver, generally the best way to perform it is to enter a 'k0' (koiogran turn zero) command, turning your ship in place :)
